src/Tickets/Flexible_Tickets/Series_Passes.php

```diff
@@ -854,4 +859,33 @@ public function filter_unticketed_count_query( string $query = null, string $pos
 
 		return $this->queries->filter_unticketed_count_query();
 	}
+
+	/**
+	 * Filter the meta box helper text for Series post type.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $text The helper text with link.
+	 * @param WP_Post $post The Post object.
+	 *
+	 * @return string
+	 */
+	public function filter_tickets_panel_list_helper_text( string $text, WP_Post $post ): string {
+		if ( Series_Post_Type::POSTTYPE != $post->post_type ) {
```

Comment:
There is a `TEC\Tickets\Flexible_Tickets\Series_Passes\Meta` class: this method should be moved there similarly to [this one.](https://github.com/the-events-calendar/event-tickets/blob/a969603260b13418cb6271bb379b92fb56db21be/src/Tickets/Flexible_Tickets/Series_Passes.php#L728-L740)